### ISSUE TITLE
3. Ontbrekende toegankelijke naam "vergrootglas/zoeken knop" (Homepage template A11y)

### ISSUE THREAD

**Issue by TessaViergever** [frameless/developer.overheid.nl#116]
### Betrokken pagina's

[Homepagina template](https://rijkshuisstijl-community-templates-3ix5516vd-nl-design-system.vercel.app/developer-overheid/homepage)

### Beschrijf het probleem

De vergrootglas-knop heeft geen toegankelijke naam.

### Relevante code

```html
`<button class="utrecht-button" type="button"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="tabler-icon tabler-icon-search "><path d="M10 10m-7 0a7 7 0 1 0 14 0a7 7 0 1 0 -14 0"></path><path d="M21 21l-6 -6"></path></svg></button>`
```

### WCAG-richtlijn

WCAG 4.1.2 Name, Role, Value Level A

### Voeg een afbeelding toe (indien relevant)

![The accessible names of the items in the main navigation are shown. The button with a magnifying glass icon has no accessible name.](https://github.com/user-attachments/assets/4337ffa0-8a74-4f1d-a8ab-43a454f32422)

### Referentienummer

TBD
